src/Models/Link.php

```diff
@@ -123,6 +126,15 @@ public function getCMSFields(): FieldList
                 $linkTypeField->setEmptyString('-- select type --');
             }
         });
+        $this->afterUpdateCMSFields(function (FieldList $fields) {
+            // Move the OpenInNew field to the bottom of the form and update its title
+            $openInNewField = $fields->dataFieldByName('OpenInNew');
+            $fields->removeByName('OpenInNew');
+            if ($this->showOpenInNewWindowField) {
+                $fields->addFieldToTab('Root.Main', $openInNewField);
+                $openInNewField->setTitle(_t(__CLASS__ . '.OPEN_IN_NEW_TITLE', 'Open in new window?'));
+            }
```

Comment:
This will throw away title changes people have made in their custom link types or extensions.

Instead of this, keep the old lines 106 and 107, and change this to:
```suggestion
            // Move the OpenInNew field to the bottom of the form and update its title
            $openInNewField = $fields->dataFieldByName('OpenInNew');
            if ($openInNewField) {
                $fields->removeByName('OpenInNew');
                $fields->addFieldToTab('Root.Main', $openInNewField);
            }
```
So:
1. we always add it by default
2. if developers don't want it, they can just remove it (no need for that protected property)
3. only remove it if it's there
4. don't change the title, because we're keeping line 107 (and also keeping any changes anyone makes in extensions or subclasses)

Reply:
Updated